src/formats/converter_factory.py

```
@@ -151,20 +153,43 @@ def convert_streaming_chunk(source_format: str, target_format: str, data: dict,
+            return converter._convert_from_responses_streaming_chunk(data)
     elif target_format == "gemini":
         if source_format == "openai" and hasattr(converter, '_convert_from_openai_streaming_chunk'):
             return converter._convert_from_openai_streaming_chunk(data)
         elif source_format == "anthropic" and hasattr(converter, '_convert_from_anthropic_streaming_chunk'):
             return converter._convert_from_anthropic_streaming_chunk(data)
+        elif source_format == "responses" and hasattr(converter, '_convert_from_responses_streaming_chunk'):
+            return converter._convert_from_responses_streaming_chunk(data)
+    elif target_format == "responses":
+        # 处理转换为responses格式的流式转换
+        if source_format == "openai" and hasattr(converter, '_convert_from_openai_streaming_chunk'):
+            # 首先转换为chat completions格式，然后转换为responses格式
```

Ruff (0.13.3)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)